firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -363,13 +364,14 @@ private PersistedInstallationEntry getPrefsWithGeneratedIdMultiProcessSafe() {
   }
 
   private String readExistingIidOrCreateFid(PersistedInstallationEntry prefs) {
-    // Check if this firebase app is the default (first initialized) instance
-    if (!firebaseApp.isDefaultApp() || !prefs.shouldAttemptMigration()) {
+    // Check if this firebase app is the default (first initialized) instance or is a chime app
+    if (!firebaseApp.getName().equals(CHIME_FIREBASE_APP_NAME)
+        && (!firebaseApp.isDefaultApp() || !prefs.shouldAttemptMigration())) {
```

Comment:
Not sure if the bracketing is right here...
I don't know what #shouldAttemptMigration does (so I might be wrong here)...
But it seems, that this should be changed to:

    if ((!firebaseApp.getName().equals(CHIME_FIREBASE_APP_NAME) && !firebaseApp.isDefaultApp())
       || !prefs.shouldAttemptMigration()) {

Reply:
Done.